packages/social-previews/src/facebook-preview/style.scss

```diff
@@ -1,155 +1,137 @@
-	font-size: 16px;
-	font-weight: 600;
-	line-height: 20px;
-	max-height: 100px;
-	transition: color 0.1s ease-in-out;
+	&.is-portrait {
+		flex-direction: row;
+	}
 }
 
-.facebook-preview__description {
-	color: #606770;
-	/* stylelint-disable-next-line declaration-property-unit-allowed-list */
-	font-size: 14px; //value in Facebook's styles
-	line-height: 20px; //value in Facebook's styles
-	overflow-y: hidden;
-}
+.facebook-preview__text {
+	position: relative;
 
-.facebook-preview__url {
-	color: #606770;
-	/* stylelint-disable-next-line declaration-property-unit-allowed-list */
-	font-size: 12px; //value in Facebook's styles
-	line-height: 11px; //value in Facebook's styles
-	text-transform: uppercase;
-	text-overflow: ellipsis;
-	white-space: nowrap;
-	overflow: hidden;
-}
+	display: flex;
+	align-items: center;
+	flex: 1;
 
-.facebook-preview__article {
-	.facebook-preview__content {
-		flex-direction: column;
-		min-width: 100%;
-	}
+	padding: 0.5rem 1rem;
 
-	.facebook-preview__image {
-		max-height: 250px;
-		display: flex;
-		justify-content: center;
-		align-items: center;
-		overflow-y: hidden;
+	background-color: $facebook-preview-text-background-color;
+	color: $facebook-preview-text-color;
 
-		& img {
-			height: auto;
-			width: 100%;
-			max-width: 527px;
-		}
-	}
+	word-break: break-word;
+}
 
-	.facebook-preview__body {
-		height: auto;
-		max-height: 100px;
-	}
+.facebook-preview__url {
+	margin-bottom: 0.25rem;
 
-	.facebook-preview__title {
-		margin-bottom: 1px;
-	}
+	color: $facebook-preview-light-text-color;
 
-	.facebook-preview__description {
-		display: -webkit-box;
-		-webkit-line-clamp: 1;
-		-webkit-box-orient: vertical;
-	}
+	/* stylelint-disable-next-line scales/font-sizes */
+	font-size: 0.8125rem; // 13px
+	line-height: 1;
+	text-transform: uppercase;
+}
 
-	.facebook-preview__url {
-		margin-bottom: 5px;
-	}
+.facebook-preview__title {
+	margin: 0.25rem 0 0.125rem;
+
+	/* stylelint-disable-next-line scales/font-sizes */
+	font-size: 0.9444rem; // 17px
+	font-weight: 600;
+	line-height: 1.18;
 }
 
-.facebook-preview__website {
-	max-height: 158px;
-	overflow: hidden;
+.facebook-preview__description {
+	color: $facebook-preview-light-text-color;
 
-	.facebook-preview__image {
-		flex-shrink: 0;
-		height: 158px;
-		width: 158px;
+	/* stylelint-disable-next-line scales/font-sizes */
+	font-size: 0.9375rem; // 15px
+	line-height: 1.33;
+}
 
-		box-sizing: border-box;
-		border: 1px solid #dadde1;
-		border-right: 0;
+.facebook-preview__image {
+	&.is-empty {
+		width: 140px;
+		height: 140px;
 
-		& img {
-			display: block;
-			font-size: $font-body-small;
-			height: auto;
-			width: 100%;
+		border-inline-end: $facebook-preview-body-border;
+	}
+
+	&.is-portrait {
+		> img {
+			width: 278px;
+			height: 430px;
 		}
+	}
 
-		&::after {
-			content: "";
-			display: block;
-			height: 100%;
+	&.is-landscape {
+		> img {
 			width: 100%;
-			background: #fff;
+			aspect-ratio: 1.91;
```

Comment:
This comes from `1200 / 630` right? Do you think it's worth adding a comment to explain?

Reply:
That's the ratio I measured on the Facebook page. The can go from 508px to 680px wide. The image follows the same ratio at any width. I'll add a comment for clarify 👍